### PR TITLE
Minor Code Error

### PR DESCRIPTION
**Pull request by Amay22**
Specified in Issues
Variable name not correct for breadcrumbs-bg
_misc.scss has .btn overloading on bootstrap

### CODE REVIEW

File: lib/assets/stylesheets/nyulibraries/_breadcrumbs.scss [hunk 1]
```diff
@@ -5,7 +5,7 @@
      -moz-border-radius: 0;
           border-radius: 0;
   color: lighten(black, 20%);
-  background-color: $breadcrumb-bg;
+  background-color: $breadcrumbs-bg;
```
Comment (barnabyalter) [inline lib/assets/stylesheets/nyulibraries/_breadcrumbs.scss]: @Amay22 The bug is actually here: https://github.com/NYULibraries/nyulibraries-assets/blob/master/lib/assets/stylesheets/nyulibraries/_variables.scss#L17

That should say `$breadcrumb-bg` to match the default variables here: https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss#L788

And then you can revert these back to `breadcrumb-bg` singular.

File: lib/assets/stylesheets/nyulibraries/_misc.scss [hunk 2]
```diff
@@ -15,11 +15,16 @@
   min-width: 200px;
 }
 
+.btn1 {
+  line-height: $line-height-large-computed;
+}
+
```
Comment (barnabyalter) [inline lib/assets/stylesheets/nyulibraries/_misc.scss]: @Amay22 I think this works the original way. If you `@extend .btn` then you are getting all the bootstrap btn styles (https://github.com/twbs/bootstrap-sass/blob/d35d4bd58708c3db82aaac7b50295ff37a2d1c83/assets/stylesheets/bootstrap/_buttons.scss#L9) but changing the line-height. 

Can you explain what your changes are doing differently?

**Comment by Amay22** [comment]
When I compile Scss to Css it gives a very weird  error and I realized that's because .btn overloads on the .btn in bootstrap.
So I changed the name of our defined .btn to .btn1 so it doesn't overload. 
Doesn't change anything but it doesn't create two instances of .btn when we compile it into one huge css file.

**Comment by Amay22** [comment]
I'll change the breadcrumbs thing to match the global variable.

**Comment by barnabyalter** [comment]
@Amay22 I see. But if you extend .btn1 then you don't get any of the attributes of .btn, which is what is intended. The desired effect is that all buttons using the btn class in our extended assets look like bootstrap buttons but have different  line heights. Can you show me what the compiled css looks like because either of the following would:

```css
.btn {
//all the button styles from bootstrap-sass
line-height: ..px;
}
```

Which is how I assume it compiles or:

```css
.btn {
//all the bootstrap-sass styles
}
.btn {
line-height: ...px;
}
```
In both cases the nature of CSS would allow  a resulting `<button class="btn" />` to look as we intended.

**Comment by Amay22** [comment]
This is complete compiled CSS.
https://github.com/Amay22/coffeeS-JS-SCSS-CSSuseGuard/blob/master/stylesheets/css/main.css

If I let it overload then it throws an error and doesn't convert to CSS

**Comment by barnabyalter** [comment]
@Amay22 Ok but you do need to structure this so that any element using the .btn class has both all the bootstrap styles and the custom line-height. And that .nyu-tip then extends this class

**Pull request closed without merging**